jquery.orgchart.js

```diff
@@ -215,4 +233,4 @@
         $appendTo.append($table);
     };
 
-})(jQuery);
+})(jQuery);
```

Comment:
I can't add this comment to your screenshot/image - but it does not look like the other ones.